docs/developer-release-signatures

```diff
@@ -0,0 +1,28 @@
+TODO: clear-sign this with the release key
+TODO: update jean-paul's information
+
+January 3, 2021
+
+Any of the following core Tahoe contributers may sign a release. Each
+release should be signed by at least two developers. They each
+independently produce a signature which are made available beside
+Tahoe releases after 1.15.0
+
+This statement is signed by the previous Tahoe release key. Any future
+such statements may be signed by it OR by any two developers (for
+example, to add or remove developers from the list).
+
+meejah
+0xC2602803128069A7
+9D5A 2BD5 688E CB88 9DEB CD3F C260 2803 1280 69A7
+https://meejah.ca/meejah.asc
+
+jean-paul calderone (exarkun)
+0xE27B085EDEAA4B1B
+96B9 C5DA B2EA 9EB6 7941 9DB7 E27B 085E DEAA 4B1B
+http://pgp.mit.edu/pks/lookup?op=get&search=0xE27B085EDEAA4B1B
```

Comment:
Confirming that this is indeed my key, although the pgp.mit.edu server continues to behave very poorly so I've also uploaded my key to https://twistedmatrix.com/~exarkun/E27B085EDEAA4B1B.asc

Reply:
For line 23, would something like one or both of these be more resilient?: 

View Key Index Details:  
http://pool.sks-keyservers.net/pks/lookup?search=0xE27B085EDEAA4B1B&fingerprint=on&hash=on&exact=on&op=vindex

or

Get Key Directly: 
http://pool.sks-keyservers.net/pks/lookup?search=0xD16C3A41949D203A&op=get

Reply:
I'm curious, just for my amateur personal edification, why use one's personal website public-key link, versus what I would think would be a much more resilient public pool keyserver? Is there a problem with keyservers, or with pools?  Thanks.

Reply:
I only have an amateur reply, I hope that's okay :)

In my brief testing while trying to get my key online somewhere, my attempts to retrieve the key from all of the keyserver URLs suggested so far failed multiple times without timeout errors before eventually succeeding.  So I'm not sure the keyservers and pools actually provide better availability.

Also, the twistedmatrix.com URL is not discoverable except by its inclusion in this document whereas if you know the key id (as given in this doc) you can always choose to visit a keyserver and try to retrieve the key if you like.  If we remove the twistedmatrix.com URL then this would leave the keyservers as essentially the *only* option.  Keeping this in mind, the total availability may end up being better by including this information.